diagnostic.md

```diff
@@ -5,56 +5,60 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of a backend, is to structure and hold data that is sent out to the client
+and is the part that communicates directly with the source of the data.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model is the part to send and retrieve data. It gets a command from the controler
+and respondes appropriately
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller is what tells the model what to do and what data to use.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+If we used the view to interperate the data, we would have to use multiple views to
+accomplish the same actions.
```

Comment:
`Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.`